### DISCUSSION TITLE
More 200 connections yesterday.today at zero

### DISCUSSION THREAD

**Discussion by realbluefish** [Chia-Network/chia-blockchain#3257] [Support]
Hello
Yesterday I went to sleep and had the GUI running with more than 200 node connections. Fully synchronized (the envy of my neighbors).
this morning when i wake up i'm out of sync. zero nodes.
keeps plotting normal but does not sync.
How can I know what has happened?
Tnx

**Comment by ESKCOBAR** [comment]
попробуйте обновить до версии 1.1.2, так же откройте порт 8444

**Comment by thentd** [comment]
这种很是正常，111、112都有这问题，105的时候 都是正常的，升级后 就不正常了，路由器开不开8444端口105都是正常的，升级就不行了，

**Comment by bobochi** [comment]
If you already have your router port forwarding 8444, when this has happened to me in the past, I rechecked to make sure my port 8444 was still open using https://www.yougetsignal.com/tools/open-ports/.  When I sent it down the first time, it reported that it was closed.  Then sending a second time reported that it was open.  Once I did that, the peers started popping in again.  Not sure of the root cause, or even if this was the solution...but it's something to recheck/try.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
